collect_app/src/test/java/org/odk/collect/android/entities/EntitiesRepositoryTest.kt

```diff
@@ -769,4 +770,119 @@ abstract class EntitiesRepositoryTest {
         assertThat(savedEntities[0].properties.size, equalTo(1))
         assertThat(savedEntities[0].properties[0].first, equalTo("prop"))
     }
+
+    @Test
+    fun `#query returns matching entities`() {
```

Comment:
I think these tests can just use the simplest possible queries (like just a basic `Query.Eq`) given that `SqlQueryTest` covers the complexity of translating the objects to actual SQL.

Reply:
Done.